publish-wrims-template/build.gradle

```diff
@@ -0,0 +1,102 @@
+
+repositories {
+    maven {
+        name = "GitHubPackages"
+        url = uri("https://maven.pkg.github.com/CentralValleyModeling/wrims")
+        credentials {
+            username = project.findProperty('githubToken') ?: System.getenv('GITHUB_TOKEN')
+            password = ''
+        }
+    }
+}
+
+configurations {
+    testFile
+}
+
+publishing {
+    publications {
+        wrimsGuiTemplateCore(MavenPublication) {
+            groupId 'gov.ca.dwr'
+            artifactId 'wrims_gui_template_core'
+            version = '1.0.0'
+            artifact(templateCoreFile) {
+                extension 'zip'
+            }
+            pom {
+                name = 'WRIMS GUI Template Core'
+                description = 'Template for WRIMS GUI'
+                url = 'https://github.com/CentralValleyModeling/wrims'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                scm {
+                    connection = 'scm:git:git://github.com/CentralValleyModeling/wrims.git'
+                    developerConnection = 'scm:git:ssh://github.com/CentralValleyModeling/wrims.git'
+                    url = 'https://github.com/CentralValleyModeling/wrims'
+                }
+            }
+        }
+//        wrimsGuiTemplate(MavenPublication) {
+//            groupId 'gov.ca.dwr'
+//            artifactId 'wrims_gui_template'
+//            version = project.findProperty('postVersion') ?: "1.0.3"
+//            artifact(zipFilePath) {
+//                extension 'zip'
+//            }
+//            pom {
+//                name = 'WRIMS GUI Template'
+//                description = 'Template for WRIMS GUI'
+//                url = 'https://github.com/CentralValleyModeling/wrims'
+//                licenses {
+//                    license {
+//                        name = 'The Apache License, Version 2.0'
+//                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+//                    }
+//                }
+//                scm {
+//                    connection = 'scm:git:git://github.com/CentralValleyModeling/wrims.git'
+//                    developerConnection = 'scm:git:ssh://github.com/CentralValleyModeling/wrims.git'
+//                    url = 'https://github.com/CentralValleyModeling/wrims'
+//                }
+//            }
+//        }
+    }
+
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/CentralValleyModeling/wrims")
+            credentials {
+                username = project.findProperty('githubToken') ?: System.getenv('GITHUB_TOKEN')
+                password = ''
+            }
+        }
+    }
+}
+
+task verifyPublishedFile {
```

Comment:
This seems unnecessary, if the publish fails it should throw an error. Were you seeing something different?

Reply:
Josh deleted subproject